apps/web-giddh/src/app/settings/integration/setting.integration.component.html

```diff
@@ -47,13 +48,19 @@ <h3 class="fs20 mrB2">MSG91</h3>
             <div class="box clearfix mb-2">
                 <h3 class="section_head">Gmail Integration</h3>
                 <div class="d-flex">
-                    <a *ngIf="!(isGmailIntegrated$ | async)" class="btn btn-primary"
-                        [href]="gmailAuthCodeUrl$ | async">Connect with
+
+                    <a *ngIf="!(isGmailIntegrated$ | async) && !isElectron " class="btn btn-primary"
```

Comment:
Avoid using multiple subscription in HTML with async pipe. This can be rewritten as:
 <ng-container *ngIf="!(isGmailIntegrated$ | async)>
    <a class="btn btn-primary" [href]="gmailAuthCodeUrl$ | async" *ngIf="!isElectron">Connect with Gmail</a>
    <a class="btn btn-primary" [href]="javascript:void(0);" (click)="gmailIntegration('google')" *ngIf="isElectron">Connect with Gmail</a>
</ng-container>